src/facade/dragonfly_connection.h

```diff
@@ -494,6 +494,8 @@ class Connection : public util::Connection {
       bool is_main_ : 1;
     };
   };
+
+  bool request_shutdown = false;
```

Comment:
there are bunch of flags two lines above like migration_in_process_ migration_enabled_ etc 
why not move there? :)